File: kotlin/math/main_11653.kt
import kotlin.math.sqrt

fun main() {
    var num = readLine()!!.toInt()
    val standard = sqrt(num.toDouble())
    var i = 2
    while (i <= standard){
        if (num % i == 0){
            println(i)
            num /= i
        } else i++
    }
    if (num != 1)
        println(num)
}

File: cpp/step/math/main_11653.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
	int num;
	cin >> num;

	for (int i = 2; i <= sqrt(num); ++i) {
		if (num % i == 0) {
			cout << i << endl;
			num /= i;
			i--;
		}
	}
	if (num != 1) {
		cout << num << endl;
	}
}
